app-modules/meeting-center/src/Http/Controllers/GoogleCalendarLoginController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Assist\MeetingCenter\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\RedirectResponse;
+use Assist\MeetingCenter\GoogleCalendarManager;
+
+class GoogleCalendarLoginController extends Controller
```

Comment:
I would recommend consolidating this and `GoogleCalendarRedirectController` into a single `MeetingCenterCalendarIntegrationController` or something like that.

See what we did with `app-modules/authorization/src/Http/Controllers/SocialiteController.php`, making use of an ENUM for the provider.